src/main/java/gov/cabinetoffice/gap/adminbackend/controllers/SchemeEditorController.java

```diff
@@ -59,4 +59,10 @@ public ResponseEntity<String> addEditorToScheme(@PathVariable final Integer sche
         return ResponseEntity.ok("Editor added to scheme successfully");
     }
 
+    @DeleteMapping(value = "/{editorId}")
+    public ResponseEntity<String> deleteSchemeEditor(@PathVariable Integer schemeId, @PathVariable Integer editorId) {
```

Comment:
Can we add an ownership check to this endpoint?

Reply:
We only want scheme owners to be able to hit this endpoint